lib/private/message_settings_update.php

```diff
@@ -70,9 +70,29 @@ function generateQueryStringText( $CurrentValues, &$BindValues, $ValueName, $New
     
         return '';
     }
-    
+
     // -----------------------------------------------------------------------------
-    
+
+    function getRequestValue( $aRequest, $sParameterName, $sValueType = "IntValue" )
+    {
+        if (is_array($aRequest) && array_key_exists($sParameterName, $aRequest))
+        {
+            return $sParameterName;
```

Comment:
function getRequestIntValue($aRequest, $sParameterName, &$aValue)
{
    $aValue = (is_array($aRequest) && array_key_exists($sParameterName, $aRequest))
        ? intval($aRequest[$sParameterName]
        : 0
}

Another function like that for strings, too.